### ISSUE TITLE
##ISSUE it showing "Device RZ8M702QHMM is not authorized. You might need to check your device for an authorization dialog."

### ISSUE THREAD

**Issue by SandyCooper** [flutter/flutter#136618]
Before two month from today's date ,  flutter doesn't have no problem I been working in two, three project by connecting my android phone wireless. There is no problem occur.

And I didn't do any projects and I didn't even connect my phone wirelessly. After two month I open my flutter project and try to connect my phone wirelessly. it shows error. When I try to solve the issue by surfing over google, stackflow and github nothing solved my issuse.

So, I come here to post my issue.  Please experts try to solve my issus or point where i need to go to solve this issue.

I have checked the flutter doctor , And no issue found.
![image](https://github.com/flutter/flutter/assets/112313667/73faaa01-c0e5-42d7-9a6f-2848a27be7ce)

Also I attached the Error showing message which is "**Device RZ8M702QHMM is not authorized.
You might need to check your device for an authorization dialog.**"
![image](https://github.com/flutter/flutter/assets/112313667/bfd78f4c-3467-4909-a8ff-4c567f1909b7)

AGAIN please rectify my issue, you experts

And any other screenshots regarding the settings, Please let me know.

**Comment by huycozy** [comment]
Hi @SandyCooper 
When opening an issue, please provide the information in text format instead of a screenshot for better readability and investigation.

Regarding the issue, have you checked [this](https://stackoverflow.com/questions/26079838/android-studio-adb-error-device-unauthorized-please-check-the-confirmat) to see if it helps?

**Comment by SandyCooper** [comment]
Hello @huycozy 

I already tried that stackflow link before I have post this issue here. There is No use. I have tried all solution that is listed.

But something I have forget to mention yesterday, That is when I tried to connect some other phone, it shows no issue and that phone connected wirelessly. I don't know what is the problem is.

**Comment by huycozy** [comment]
It looks like an issue with your specific device setting. Have you tried [this](https://stackoverflow.com/a/60650449/5921933) solution as well?

**Comment by SandyCooper** [comment]
Hello @huycozy 

There something funny happened. Yesterday I said know, that I have tried some other phone there is no problem occurred.

Then today I thought to connect my phone for the last time. And there we go, No problem occur and my phone connect successfully. I'm Speechless because I tried to connect my phone for past four days all the time shows error. Yesterday I connect other phone and I have tried my phone today it is connected.

I think my phone got jealous with the other phone and stopped playing with me 😂😂

I hope this is the solution for this kind of issue. "**Try connect other phone successfully and then try with your phone**" . If there is any technical reason for this, kindly share here.

And @huycozy  thank you for your time to suggesting some solutions.
